lib/automation/report_runner.rb

```diff
@@ -35,25 +35,30 @@ def self.run!(**args)
 
       def initialize(
         sender_username:,
-        receivers:,
-        title:,
         model:,
-        category_ids:,
-        tags:,
-        allow_secure_categories:,
-        debug_mode:,
         sample_size:,
         instructions:,
+        tokens_per_post:,
         days:,
         offset:,
-        priority_group_id:,
-        tokens_per_post:
+        receivers: nil,
+        topic_id: nil,
+        title: nil,
+        category_ids: nil,
+        tags: nil,
+        priority_group_id: nil,
+        allow_secure_categories: false,
+        debug_mode: false
       )
         @sender = User.find_by(username: sender_username)
         @receivers = User.where(username: receivers)
-        @email_receivers = receivers.filter { |r| r.include? "@" }
-        @title = title
-
+        @email_receivers = receivers&.filter { |r| r.include? "@" }
```

Comment:
IIRC I said this on the last PR, but using `EmailAddressValidator.valid_value?(r)` would be better for filtering here

Reply:
I get that ... but then there is even more code required... eg `sam@sam` is not a valid email and certainly not a valid user. 

So in that case we would need to take that ... and raise an exception that you typoed an email... which is fine, but complicates the code

Reply:
Ah I see, I hadn't seen the AI UI for this yet -- which doesn't allow entering a bad email address :+1: